### PR TITLE
Issues/1740 live tailing order

### PR DESCRIPTION
**Pull request by Mgrdich**
- [ ] **Breaking change?** (if so, please describe the impact and migration path for existing application instances)

**What changes did you make?** (Give an overview)
minor message addition order during live mode add from the top of table otherwise normal

**Is there anything you'd like reviewers to focus on?**

**How Has This Been Tested?** (put an "x" (case-sensitive!) next to an item)

- [ ] No need to
- [ ] Manually (please, describe, if necessary)
- [x] Unit checks
- [ ] Integration checks
- [x] Covered by existing automation

**Checklist** (put an "x" (case-sensitive!) next to all the items, otherwise the build will fail)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (e.g. **ENVIRONMENT VARIABLES**)
- [x] My changes generate no new warnings (e.g. Sonar is happy)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged

Check out [Contributing](https://github.com/provectus/kafka-ui/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/provectus/kafka-ui/blob/master/CODE-OF-CONDUCT.md)

**A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Comment by Mgrdich** [comment]
in Filters i did not add any test suites , cause they have to be added in a separate issues maybe.

**Review by workshur — APPROVED** [review]

**Review by rAzizbekyan — APPROVED** [review]

**Review by Haarolean — CHANGES_REQUESTED** [review]
Currently 'oldest first' is working incorrectly

File: kafka-ui-react-app/src/redux/reducers/topicMessages/__test__/fixtures.ts [hunk 1]
```diff
@@ -15,6 +15,17 @@ export const topicMessagePayload: TopicMessage = {
...
+export const topicMessagePayloadV2: TopicMessage = {
+  partition: 28,
+  offset: 88,
+  timestamp: new Date('2021-07-21T23:25:14.865Z'),
+  timestampType: TopicMessageTimestampTypeEnum.CREATE_TIME,
+  key: 'schema-registry',
+  headers: {},
+  content:
+    '{"host":"schemaregistry1","port":8085,"master_eligibility":true,"scheme":"http","version":1}',
+};
```
Comment (simonyandev) [inline kafka-ui-react-app/src/redux/reducers/topicMessages/__test__/fixtures.ts:27]: As I can see, the above 7 lines are the exact same thing that appear in topicMessagePayload declared above. Can we reduce code duplication by storing the repeated information in a separate variable and using it by spreading at the end?

**Review by simonyandev — CHANGES_REQUESTED** [review]

File: kafka-ui-react-app/src/redux/reducers/topicMessages/__test__/fixtures.ts (hunk 1 above)
Comment (Mgrdich) [reply kafka-ui-react-app/src/redux/reducers/topicMessages/__test__/fixtures.ts:27]: Sure.

**Review by simonyandev — APPROVED** [review]

**Pull request merged**